lighthouse-core/test/fraggle-rock/api-test-pptr.js

```diff
@@ -130,7 +130,7 @@ describe('Fraggle Rock API', () => {
         notApplicableAudits,
       } = getAuditsBreakdown(lhr);
       // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
-      expect(auditResults.length).toMatchInlineSnapshot(`59`);
```

Comment:
I don't understand why this went down. There are no explicit filters for the timespan tests right?

Reply:
ah sorry forgot to comment, this went down because we now filter the config by default of any audits that aren't included in any categories (except full-page-screenshot). the `manual` audits in a11y were all trimmed because we had logic that didn't display them, but nothing that removed their JSON from the payload.